common/src/main/java/com/linecorp/centraldogma/internal/Util.java

```diff
@@ -179,6 +179,19 @@ public static String validateEmailAddress(String emailAddr, String paramName) {
                 " (expected: " + EMAIL_PATTERN.pattern() + " or IP address domain)");
     }
 
+    public static String ensureEmailAddress(String emailAddr, String paramName) {
+        requireNonNull(emailAddr, paramName);
+        if (isValidEmailAddress(emailAddr)) {
+            return emailAddr;
+        }
+        return emailAddr + "@localhost.localdomain";
```

Comment:
Need to raise an exception if the result is not a valid email address yet.

Reply:
I think I decided bad name for this method, so I changed it to `toEmailAddress` which means converting an argument `emailAddr` to an email address if it's not.